man/docker-run.1.md

```diff
@@ -374,6 +375,9 @@ its root filesystem mounted as read only prohibiting any writes.
     "label:level:LEVEL" : Set the label level for the container
     "label:disable"     : Turn off label confinement for the container
 
+**--stop-signal**=15
+  Signal to stop a container. Default is 15.
```

Comment:
*Anticipating docs review a bit.* Can we say `SIGTERM` in the docs rather than 15?